stable/kafka-manager/README.md

```diff
@@ -53,6 +53,15 @@ Parameter | Description | Default
 `basicAuth.enabled` | If ture, enable basic authentication | `false`
 `basicAuth.username` | Username for basic auth | `admin`
 `basicAuth.password` | Paswword for basic auth | `""`
+`basicAuth.ldap.enabled` | If true, enable LDAP authentication | `false`
```

Comment:
Should ldap be under “basicAuth”?

Reply:
CMAK thinks so : https://github.com/yahoo/CMAK/blob/1.3.3.23/conf/application.conf#L39

Also, the LDAP backend still uses HTTP Basic Authentication.

Reply:
I actually wanted to rename the property to be `basicAuthentication` instead of `basicAuth` to match the configuration name that CMAK expects. I did not want to break everyone else's workflow.